dd-sdk-android/src/test/kotlin/com/datadog/android/rum/assertj/ActionEventAssert.kt

```diff
@@ -207,6 +207,30 @@ internal class ActionEventAssert(actual: ActionEvent) :
         return this
     }
 
+    fun isEqualTo(expectedActionEvent: ActionEvent): ActionEventAssert {
```

Comment:
`ActionEvent` and all generated classes are `data class`, meaning that `equals` already checks the equality field by field. So basically you can just let it use the default implementation.

Reply:
It didn't work this way...problem is that if that Data object has a Data object field it will try to assert them as reference address. At least this is what I experienced when I used it that way. I will try one more time to make sure I was not missing something.

Reply:
I managed to use the default implementation by excluding the `dd` property for ErrorEvent and ActionEvent as discussed.